pyquil/api/qpu.py

```diff
@@ -154,7 +154,7 @@ def __init__(self, device=None, async_endpoint='https://job.rigetti.com/beta', a
         self.ping_time = ping_time
         self.status_time = status_time
 
-    def run(self, quil_program, classical_addresses, trials=1, needs_compilation=True, isa=None):
+    def run(self, quil_program, classical_addresses=[], trials=1, needs_compilation=True, isa=None):
```

Comment:
No mutable default arguments. Do classical_addresses=None and then `if not classical_addresses` below.

http://docs.python-guide.org/en/latest/writing/gotchas/#mutable-default-arguments